wire-ios-sync-engine/Source/SessionManager/BackendEnvironmentProvider+Cookie.swift

```diff
@@ -25,12 +25,10 @@ extension BackendEnvironmentProvider {
     }
 
     public func isAuthenticated(_ account: Account) -> Bool {
-        let cookieStorage = self.cookieStorage(for: account)
-
-        if let expirationDate = cookieStorage.authenticationCookieExpirationDate {
-            return expirationDate.timeIntervalSinceNow > 0
-        } else {
-            return cookieStorage.authenticationCookieData != nil
+        guard let expirationDate = cookieStorage(for: account).authenticationCookieExpirationDate else {
+            return false
         }
+
```

Comment:
I removed the else branch here because my changes would use `hasAuthenticationCookie` which implicitly check that there exists an expiration date.